frontend/packages/kubevirt-plugin/src/selectors/vm-template/advanced.ts

```diff
@@ -49,6 +50,17 @@ export const getTemplatesLabelValues = (templates: TemplateKind[], label: string
 export const getTemplateFlavors = (templates: TemplateKind[]) =>
   getTemplatesLabelValues(templates, TEMPLATE_FLAVOR_LABEL);
 
+export const getTemplateHostname = (template: TemplateKind) => {
+  const asVM = selectVM(template);
+  return (
```

Comment:
can we rather use 

```ts
_.get(vm, 'spec.template.spec') && vm.spec.template.spec.hostname
```
then building pyramids like this one?

Reply:
AFAIK @mareklibra  said not to use `_.get` anymore

Reply:
I think he meant plain use of `_.get` in selectors, but IMHO this usage should be ok